bin/loop.py

```diff
@@ -66,27 +66,9 @@ def run(self):
         """
         actually run the function
         """
-        if self.ingest is True:
```

Comment:
Please also remove the parser.add_argument lines associated with ingest on line 50